Fix failures in Manage Packages tests

These failing tests were disabled in https://github.com/microsoft/azuredatastudio/pull/8876. Opening this issue to track fixing them.

1. Manage Package Providers Pip getPackageOverview should return package info successfully:
AssertionError: expected [Promise] to be fulfilled with Object {
name: 'name',
versions: Array [ '0.0.1', '0.0.2' ],
summary: 'summary'
}, but it was rejected with SyntaxError { message: 'Unexpected token i in JSON at position 7' }
at Context. (C:\Users\Cory\Source\azuredatastudio\extensions\notebook\src\test\managePackages\localPackageManageProvider.test.ts:166:3)

2. Manage Package Providers Conda getPackageOverview should return package info successfully:
AssertionError: expected undefined to equal Object {
name: 'name',
versions: Array [ '0.0.1', '0.0.2' ],
summary: undefined
}
at Context. (C:\Users\Cory\Source\azuredatastudio\extensions\notebook\src\test\managePackages\localPackageManageProvider.test.ts:183:3)

3. Manage Packages Init should throw exception not given valid default location for single location mode:
AssertionError: expected [Promise] to be rejected
at Context. (C:\Users\Cory\Source\azuredatastudio\extensions\notebook\src\test\managePackages\managePackagesDialogModel.test.ts:86:3)

4. Manage Packages current provider should install and uninstall packages successfully:
AssertionError: expected [Promise] to be rejected
at Context. (C:\Users\Cory\Source\azuredatastudio\extensions\notebook\src\test\managePackages\managePackagesDialogModel.test.ts:318:3)

--- a/extensions/notebook/src/test/managePackages/localPackageManageProvider.test.ts
+++ b/extensions/notebook/src/test/managePackages/localPackageManageProvider.test.ts
@@ -153,12 +153,10 @@ describe('Manage Package Providers', () => {
 		should.equal(await provider.canUseProvider(), true);
 	});
 
-	/* Tests disabled. Tracking issue: https://github.com/microsoft/azuredatastudio/issues/8877
 	it('Pip getPackageOverview should return package info successfully', async function (): Promise<void> {
 		let testContext = createContext();
 		testContext.piPyClient.fetchPypiPackage = (packageName) => {
-			return Promise.resolve(`
-			"{"info":{"summary":"package summary"}, "releases":{"0.0.1":[{"comment_text":""}], "0.0.2":[{"comment_text":""}]}"`);
+			return Promise.resolve(`{"info":{"summary":"package summary"}, "releases":{"0.0.1":[{"comment_text":""}], "0.0.2":[{"comment_text":""}]}}`);
 		};
 		let serverInstallation = createJupyterServerInstallation(testContext);
 		let client = createPipyClient(testContext);
@@ -166,16 +164,15 @@ describe('Manage Package Providers', () => {
 
 		await should(provider.getPackageOverview('name')).resolvedWith({
 			name: 'name',
-			versions: ['0.0.1', '0.0.2'],
-			summary: 'summary'
+			versions: ['0.0.2', '0.0.1'],
+			summary: 'package summary'
 		});
 	});
 
 	it('Conda getPackageOverview should return package info successfully', async function (): Promise<void> {
 		let testContext = createContext();
 		testContext.serverInstallation.executeBufferedCommand = (command) => {
-			return Promise.resolve(`
-			"{"name":[{"version":"0.0.1"}, {"version":"0.0.2}]"`);
+			return Promise.resolve(`{"name":[{"version":"0.0.1"}, {"version":"0.0.2"}]}`);
 		};
 		let serverInstallation = createJupyterServerInstallation(testContext);
 
@@ -183,11 +180,10 @@ describe('Manage Package Providers', () => {
 
 		await should(provider.getPackageOverview('name')).resolvedWith({
 			name: 'name',
-			versions: ['0.0.1', '0.0.2'],
+			versions: ['0.0.2', '0.0.1'],
 			summary: undefined
 		});
 	});
-	*/
 
 	function createContext(): TestContext {
 		return {
@@ -222,6 +218,7 @@ describe('Manage Package Providers', () => {
 		mockInstance.setup(x => x.uninstallPipPackages(TypeMoq.It.isAny())).returns((packages, useMinVersion) => testContext.serverInstallation.uninstallPipPackages(packages));
 		mockInstance.setup(x => x.getInstalledPipPackages()).returns(() => testContext.serverInstallation.getInstalledPipPackages());
 		mockInstance.setup(x => x.getInstalledCondaPackages()).returns(() => testContext.serverInstallation.getInstalledCondaPackages());
+		mockInstance.setup(x => x.executeBufferedCommand(TypeMoq.It.isAny())).returns((command) => testContext.serverInstallation.executeBufferedCommand(command));
 		mockInstance.setup(x => x.usingConda).returns(() => testContext.serverInstallation.usingConda);
 		return mockInstance;
 	}

--- a/extensions/notebook/src/test/managePackages/managePackagesDialogModel.test.ts
+++ b/extensions/notebook/src/test/managePackages/managePackagesDialogModel.test.ts
@@ -275,7 +275,6 @@ describe('Manage Packages', () => {
 		await should(model.uninstallPackages(TypeMoq.It.isAny())).rejected();
 	});
 
-	/* Test disabled. Tracking issue: https://github.com/microsoft/azuredatastudio/issues/8877
 	it('current provider should install and uninstall packages successfully', async function (): Promise<void> {
 		let testContext1 = createContext();
 		testContext1.provider.providerId = 'providerId1';
@@ -318,9 +317,8 @@ describe('Manage Packages', () => {
 		await should(model.installPackages(packages)).resolved();
 		await should(model.uninstallPackages(packages)).resolved();
 		await should(model.getPackageOverview('p1')).resolved();
-		await should(model.getLocationTitle()).rejectedWith('location title 2');
+		await should(model.getLocationTitle()).resolvedWith('location title 2');
 	});
-	*/
 
 	function createContext(): TestContext {
 		return {

